game/templates/game/board.html

```
@@ -266,6 +266,13 @@
             </div>
 
             <div class="status-bar" id="statusBar" style="height: 22px; box-sizing: border-box;">
+                <div id="replayControls" class="replay-controls hidden">
+                    <button id="firstReplayBtn">⏮</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)